### DISCUSSION TITLE
Question about clickhouse installation package (.rpm and .deb)

### DISCUSSION THREAD

**Discussion by 1261385937** [ClickHouse/ClickHouse#45392] [Q&A] [answered]
XXX.rpm for Redhat、Centos、Fedora and so on ?
XXX.deb for Debian、Ubuntu and so on ?

If the binary built on Centos and get the .rpm, should I rebuilt the binary for Fedora ?
Or the XXX.rpm can work well on both Redhat、Centos、Fedora and so on ?

thank you for reply

**Comment by santrancisco** [comment, marked as answer]
For more information, you can check instruction for ClickHouse installation [here](https://clickhouse.com/docs/en/install/). 

AFAIK, Centos, Fedora, Redhat can interchangeably use same RPM package - Note that it must be compiled for the system architect that you are on (x86_64 vs arm64 ...)
